steamed_hams.sh

```diff
@@ -6,17 +6,17 @@
 
 # Title Header.
 echo ""
-echo "Project Borealis Booter" 
+echo "Project Borealis Booter"
 echo "v2.3-Alpha Season 1 Episode 3"
 echo "-----------------------------------------------------------------------------------"
 
```

Comment:
I think this script should be totally rewritten for actual radar operation. It needs to be headless so that we can log in and see whats happening remotely. This script is just useful for me right now to quickly start up the system for testing. I think I will just leave this for now.